<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForDelete</class>
 <widget class="QDialog" name="ForDelete">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background : rgb(255, 71, 9)</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../lab04_small.jpg</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteCondition">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background : rgb(255, 255, 255) ; 
color : rgb(255, 71, 9);
border-radius : 15;

</string>
   </property>
   <property name="text">
    <string>Delete by condition</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteIndex">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background : rgb(255, 255, 255) ; 
color : rgb(255, 71, 9);
border-radius : 15;

</string>
   </property>
   <property name="text">
    <string>Delete by index </string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteAll">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background : rgb(255, 255, 255) ; 
color : rgb(255, 71, 9);
border-radius : 15;

</string>
   </property>
   <property name="text">
    <string>Delete all</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IndexEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background : rgb(255, 255, 255)</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Enter index</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OpenButton_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background : rgb(255, 255, 255) ; 
color : rgb(255, 71, 9);
border-radius : 15;

</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/User/Downloads/imgonline-com-ua-Resize-7vPfUbXbO1ug.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color : rgb(170, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
